shipment_advice/views/shipment_advice.xml

```diff
@@ -357,4 +357,20 @@
         parent="stock.menu_stock_warehouse_mgmt"
         action="shipment_advice_action"
     />
+    <!-- Shipment Advice - Print Delivery Slip -->
+    <record id="action_print_batch_delivery_slip" model="ir.actions.server">
+        <field name="name">Print Delivery Slip</field>
+        <field name="model_id" ref="shipment_advice.model_shipment_advice" />
+        <field
+            name="binding_model_id"
+            ref="shipment_advice.model_shipment_advice"
+        />
+        <field name="state">code</field>
+        <field
+            name="code"
+        >picking_type = env.ref("stock.picking_type_out")
+pickings = records.mapped('loaded_picking_ids').filtered(lambda p: p.picking_type_id == picking_type and p.state != 'cancel')
```

Comment:
better creating a method for that to allow for easy overriding if needed

Reply:
hi @nilshamerlinck the PR has been updated